tests/login.yaml

```
@@ -0,0 +1,29 @@
+- waitForAnimationToEnd
+
+# insert login data
+- tapOn: "Enter your email"
+- waitForAnimationToEnd
+- inputText: "osoba3"
+- inputText: "@test.hr"
+- tapOn: "Enter your password"
+- waitForAnimationToEnd
+- inputText: "Pa$$w0rd"
+
+# log in 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)